force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -64,6 +71,13 @@
         <shortDescription>Delete</shortDescription>
         <value>Delete</value>
     </labels>
+    <labels>
+        <fullName>End_Time</fullName>
```

Comment:
Since these should be fields and field labels and this is just temporary, do we need to add these labels or can we just hardcode them in the sample data for now?

Reply:
@jjbennett , This is where I was confused since if we use datetime datatype then we will have to split up the date and time on the datatable since it would be confusing for the users when we display start date time. Let me know your thoughts. It will be easy to pull the labels out and hard code the value.

Reply:
After syncing up we decided to leave these for now until we figure out the definitive plan in regards to date / time fields.